subprojects/gradle/cd/src/main/kotlin/com/avito/ci/steps/TestSummaryPluginBuildStep.kt

```diff
@@ -42,7 +42,8 @@ public abstract class TestSummaryPluginBuildStep(context: String, name: String)
 
         if (useImpactAnalysis && !project.internalModule.isModified()) return
 
-        val instrumentationTask = project.tasks.instrumentationTask(configuration)
+        // flavors not supported here
```

Comment:
And what does it mean? What should I consider looking at this comment?

Reply:
BuildStep will be removed, it's a reminder that CiPlugin doesn't know about flavors